complete-application/package.json

```diff
@@ -10,7 +10,7 @@
     "type-check": "vue-tsc --noEmit -p tsconfig.app.json --composite false"
   },
   "dependencies": {
-    "@fusionauth/vue-sdk": "1.0.0",
+    "@fusionauth/vue-sdk": "^1.0.0",
```

Comment:
Lets change this to ^1.1.0 so we enforce that as the minimum

Reply:
thanks for pointing that out!